src/test/java/com/qaprosoft/carina/demo/WebSampleTest.java

```diff
@@ -112,4 +112,22 @@ public void testNewsSearch() {
         }
     }
 
+    @Test
+    public void testBaseHeaderElements(){
+        HomePage homePage = new HomePage(getDriver());
+        homePage.open();
+        Assert.assertTrue(homePage.isPageOpened(), "Home page was not opened!");
+
+        Assert.assertTrue(homePage.getHeader().checkIfBurgerMenuPresent(), "Burger menu was not found!");
```

Comment:
Good.But let's create one method like(validateHeaderBasicElements), which include all this checks.It will help us to avoid "boilerplate code" in our test.I also recommend you to add  @MethodOwner(owner = "your name")
    @Test(description = "What this test is created for")